app/src/lib/app-state.ts

```diff
@@ -353,7 +353,7 @@ export function isRebaseConflictState(
 }
 
 /**
- * Conflicts can occur during a rebase, merge, or cherry pick.
+ * Conflicts can occur during a rebase, merge, or cherry pick
```

Comment:
Nit (and LOL): I liked the `.` 😂  (I guess you deleted that `.` by accident, but if not feel free to ignore my dumb commit 🤣 )

Reply:
Lol.. I guess that period just made me mad one day.. 😄